PivotCalibration/Logic/vtkSlicerPivotCalibrationLogic.h

```diff
@@ -91,6 +95,13 @@ class VTK_SLICER_PIVOTCALIBRATION_MODULE_LOGIC_EXPORT vtkSlicerPivotCalibrationL
   // Computes the maximum orientation difference in degrees between the first tool transformation
   // and all the others. Used for determining if there was enough variation in the input data.
   double GetMaximumToolOrientationDifferenceDeg();
+
+  // Check if the shaft direction is ok, and flip it if necessary
```

Comment:
"flip it if necessary" is not very clear. Could you describe what will be the direction after calling this method (or refer to the place where it is specified)?